src/com/pessolutions/diary/MyFrame.java

```diff
@@ -0,0 +1,65 @@
+package com.pessolutions.diary;
+
+import javax.swing.*;
+import javax.swing.plaf.basic.BasicMenuBarUI;
+import java.awt.*;
+
+public class MyFrame  {
+
+    public static void frameCreate(){
+
+        // Creation of frame and its elements
+
+        ImageIcon iconFrame = new ImageIcon("D:\\Pictures\\athena-power-symbol-achilles-athens-greek-god.png");
+
+        JFrame testFrame = new JFrame("Parser v.0.1");
+
+        JLabel testLabel = new JLabel("Main menu");
+
+        JMenuBar testMenu = new JMenuBar();
+
+        //MenuElement
+
+
+        // Frame configuration
+
+        testFrame.setLayout(new BorderLayout());
+
+        testFrame.setSize(700,700);
```

Comment:
Do not use fixed size or place them into separate class as constants. It's better to rely on user's window resolution.